vm/ubpf_jit_x86_64.c

```diff
@@ -82,15 +117,17 @@ ubpf_set_register_offset(int x)
 static int
 translate(struct ubpf_vm *vm, struct jit_state *state, char **errmsg)
 {
-    emit_push(state, RBP);
-    emit_push(state, RBX);
-    emit_push(state, R13);
-    emit_push(state, R14);
-    emit_push(state, R15);
-
-    /* Move rdi into register 1 */
-    if (map_register(1) != RDI) {
-        emit_mov(state, RDI, map_register(1));
+    int i;
+
+    // Save platform non-volatile registers
```

Comment:
```suggestion
    /* Save platform non-volatile registers */
```
ubpf just uses classic C comment style